diff --git a/javelin/steno_key_code_emitter.cc b/javelin/steno_key_code_emitter.cc
index aa3bc5c..1c76a89 100644
--- a/javelin/steno_key_code_emitter.cc
+++ b/javelin/steno_key_code_emitter.cc
@@ -97,7 +97,7 @@ const uint16_t StenoKeyCodeEmitter::EmitterContext::ASCII_KEY_CODES[] = {
   0,           0,            0,              KeyCode::ESC, 0,     0,     0,     0,
 
   //              !               "               #               $               %               &               '
-  KeyCode::SPACE, LS|KeyCode::_1, LS|KeyCode::APOS,LS|KeyCode::_3,   LS|KeyCode::_4,   LS|KeyCode::_5,   LS|KeyCode::_7,   KeyCode::APOSTROPHE,
+  KeyCode::SPACE, LS|KeyCode::_1, LS|KeyCode::_2,KeyCode::HASH_TILDE,LS|KeyCode::_4,   LS|KeyCode::_5,   LS|KeyCode::_7,   KeyCode::APOSTROPHE,
   // (            )               *               +               ,               -               .               /
   LS|KeyCode::_9, LS|KeyCode::_0, LS|KeyCode::_8, LS|KeyCode::EQUAL, KeyCode::COMMA,  KeyCode::MINUS,  KeyCode::DOT,   KeyCode::SLASH,
   // 0            1               2               3               4               5               6               7
@@ -105,13 +105,13 @@ const uint16_t StenoKeyCodeEmitter::EmitterContext::ASCII_KEY_CODES[] = {
   // 8            9               :               ;               <               =               >               ?
   KeyCode::_8,    KeyCode::_9,    LS|KeyCode::SCN,KeyCode::SCN,   LS|KeyCode::COMMA,KeyCode::EQUAL,   LS|KeyCode::DOT, LS|KeyCode::SLASH,
   // @            A               B               C               D               E               F               G
-  LS|KeyCode::_2, LS|KeyCode::A,  LS|KeyCode::B,  LS|KeyCode::C,  LS|KeyCode::D,   LS|KeyCode::E,   LS|KeyCode::F,   LS|KeyCode::G,
+  LS|KeyCode::APOS, LS|KeyCode::A,  LS|KeyCode::B,  LS|KeyCode::C,  LS|KeyCode::D,   LS|KeyCode::E,   LS|KeyCode::F,   LS|KeyCode::G,
   // H            I               J               K               L               M               N               O
   LS|KeyCode::H,  LS|KeyCode::I,  LS|KeyCode::J,  LS|KeyCode::K,  LS|KeyCode::L,   LS|KeyCode::M,   LS|KeyCode::N,   LS|KeyCode::O,
   // P            Q               R               S               T               U               V               W
   LS|KeyCode::P,  LS|KeyCode::Q,  LS|KeyCode::R,  LS|KeyCode::S,  LS|KeyCode::T,   LS|KeyCode::U,   LS|KeyCode::V,   LS|KeyCode::W,
   // X            Y               Z               [               \               ]               ^               _
-  LS|KeyCode::X,  LS|KeyCode::Y,  LS|KeyCode::Z,  KeyCode::L_BRACKET,  KeyCode::BACKSLASH,  KeyCode::R_BRACKET,  LS|KeyCode::_6,   LS|KeyCode::MINUS,
+  LS|KeyCode::X,  LS|KeyCode::Y,  LS|KeyCode::Z,  KeyCode::L_BRACKET,  KeyCode::BACKSLASH_PIPE,  KeyCode::R_BRACKET,  LS|KeyCode::_6,   LS|KeyCode::MINUS,
   // `            a               b               c               d               e               f               g
   KeyCode::GRAVE, KeyCode::A,     KeyCode::B,     KeyCode::C,     KeyCode::D,     KeyCode::E,     KeyCode::F,     KeyCode::G,
   // h            i               j               k               l               m               n               o
@@ -119,7 +119,7 @@ const uint16_t StenoKeyCodeEmitter::EmitterContext::ASCII_KEY_CODES[] = {
   // p            q               r               s               t               u               v               w
   KeyCode::P,     KeyCode::Q,     KeyCode::R,     KeyCode::S,     KeyCode::T,     KeyCode::U,     KeyCode::V,     KeyCode::W,
   // x            y               z               {               |               }               ~               DEL
-  KeyCode::X,     KeyCode::Y,     KeyCode::Z,     LS|KeyCode::L_BRACKET,LS|KeyCode::BACKSLASH,LS|KeyCode::R_BRACKET,LS|KeyCode::GRAVE, 0,
+  KeyCode::X,     KeyCode::Y,     KeyCode::Z,     LS|KeyCode::L_BRACKET,LS|KeyCode::BACKSLASH_PIPE,LS|KeyCode::R_BRACKET,LS|KeyCode::HASH_TILDE, 0,
 };
 #undef S
 // spellchecker: enable
